packages/webview_flutter/webview_flutter_platform_interface/lib/src/v4/webview_platform.dart

```diff
@@ -0,0 +1,87 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
+import 'package:plugin_platform_interface/plugin_platform_interface.dart';
+import 'package:webview_flutter_platform_interface/src/v4/types/types.dart';
+import 'package:webview_flutter_platform_interface/src/v4/webview_widget_delegate.dart';
+
+import 'navigation_callback_delegate.dart';
+import 'webview_controller_delegate.dart';
+import 'webview_cookie_manager_delegate.dart';
+
+export 'types/types.dart';
+
+/// Interface for a platform implementation of a WebView.
+abstract class WebViewPlatform extends PlatformInterface {
```

Comment:
Could you make every `create...Delegate` take a `CreationParams` class? As in `createMyClassDelegate(MyClassCreationParams params)`.

Some of the params classes would be empty, but it protects us from breaking changes later if any platform implementation needs additional parameters. Which I think is likely.

Reply:
Added `...CreationParams` classes and updated the `...Delegate` classes to accept the matching `...CreationParams` class as part of their constructors.